.env.example

```
@@ -14,10 +14,10 @@ REACT_APP_TESTNET_LINK=
 REACT_APP_DEVNET_LINK=
 REACT_APP_AMM_LINK=
 REACT_APP_NFTSANDBOX_LINK=
-REACT_APP_SIDECHAIN_LINK=
+REACT_APP_CUSTOMNETWORK_LINK=
```

I'd recommend splitting up the words here like you do elsewhere to preserve the intuitive spelling.
```suggestion
REACT_APP_CUSTOM_NETWORK_LINK=
```

---

I kept them smushed together because `NFTSANDBOX` does the same. It was more intuitive to me to have the format be `REACT_APP_[network type]_LINK`

---

Stylistic difference - I don't have much of a preference (Although I did just notice that you could read the NFTSANDBOX as NFTS AND BOX which I find funny :P )